Movie/MovieMVVMTests/FileManagerTests.swift

```diff
@@ -0,0 +1,37 @@
+// FileManagerTests.swift
+// Copyright © Suprun D. All rights reserved.
+
+//
+//  FileManagerTests.swift
+//  MovieMVVMTests
+//
+//  Created by Дмитрий Супрун on 13.02.23.
+//
+@testable import Movie_MVVM
+
+import Foundation
+import XCTest
+
+/// File ManagerTests
+final class FileManagerTests: XCTestCase {
+    // MARK: - Private Properties
+
+    private var fileManager: FileServiceProtocol?
+
+    // MARK: - Public Methods
+
+    override func setUpWithError() throws {
+        fileManager = FileService()
+    }
+
+    override func tearDownWithError() throws {
+        fileManager = nil
+    }
+
+    func testSaveFetch() throws {
```

Comment:
```suggestion
    func testSaveFetch() {
```
или обрабатывай исключения